src/ol/layer/Group.js

```diff
@@ -68,6 +68,9 @@ export class GroupEvent extends Event {
  * be visible.
  * @property {Array<import("./Base.js").default>|Collection<import("./Base.js").default>} [layers] Child layers.
  * @property {Object<string, *>} [properties] Arbitrary observable properties. Can be accessed with `#get()` and `#set()`.
+ * @property {boolean} [startDeclutterGroup] Signals to the renderer to start a new declutter group, which causes previous vector(tile)
+ * layers with the declutter option set true to declutter before rendering the current layer. May be set on multiple layers to manage
+ * multiple declutter efforts.
```

Comment:
The new option won't work on layer groups (because they are not included in the `layerStatesArray`), so it should not be added here.